index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device=width">
+        <title>Weather Report</title>
+        <link rel="preconnect" href="https://fonts.gstatic.com">
+        <link href="https://fonts.googleapis.com/css2?family=Raleway&display=swap" rel="stylesheet">    
+        <link href="styles/index.css" rel="stylesheet" type="text/css"/>
+    </head>
+
+    <body>
+        <header class= "header__header">
```

Comment:
I'm curious about the double underscores in the classes! did you see this somewhere?

Reply:
EDIT: disregard my confusion on this Libby! I didn’t realize simon’s demo used the BEM naming convention until after I sent this! 

This looks good :)